Mathlib/Algebra/Order/Nonneg/Module.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2023 Apurva Nakade. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Apurva Nakade
+-/
+import Mathlib.Algebra.Order.Nonneg.Ring
+import Mathlib.Algebra.Module.Basic
+
+
+/-!
+# Modules over nonnegative elements
+
+This file defines instances and prove some properties about modules over nonnegative elements
+`{c : 𝕜 // 0 ≤ c}` of an arbitrary `OrderedSemiring 𝕜`.
+
+These instances are useful for working with `ConvexCone`.
+
+-/
+
+variable {𝕜 E : Type*}
+
+variable [OrderedSemiring 𝕜]
+variable [AddCommMonoid E] [Module 𝕜 E]
+
+namespace Nonneg
+
+/-- A module over and ordered Semiring is also a module over just the non-negative scalars. -/
+instance instModule : Module { c : 𝕜 // 0 ≤ c } E :=
```

Comment:
Spaces around parentheses were eagerly added by mathport, but we don't want them.
```suggestion
instance instModule : Module {c : 𝕜 // 0 ≤ c} E :=
```